website/docs/r/tm_org_vdc.html.markdown

```diff
@@ -0,0 +1,92 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_tm_org_vdc"
+sidebar_current: "docs-vcd-resource-tm-org-vdc"
+description: |-
+  Provides a resource to manage VMware Cloud Foundation Tenant Manager Organization VDC.
+---
+
+# vcd\_tm\_org\_vdc
+
+Provides a resource to manage VMware Cloud Foundation Tenant Manager Organization VDC.
+
+## Example Usage
+
+```hcl
+data "vcd_vcenter" "vc" {
+  name = "my-vcenter"
+}
+
+data "vcd_tm_supervisor" "supervisor" {
+  name       = "my-supervisor"
+  vcenter_id = vcd_vcenter.vc.id
+}
+
+data "vcd_tm_region_zone" "one" {
+  region_id = vcd_tm_region.region.id
```

Comment:
Thinking whether we should provide a data source for region, so the example is self-complete. Alternatively, add a comment that region should be created by the vcd_tm_region resource or read by data source.

Reply:
Added data source and updated both references